kirstie/resume.md

```diff
@@ -18,6 +18,7 @@ As part of the 2016 Mozilla Fellows for Science program, I completed/participate
 
 Event | Location | Date | Role | Invited by
 ----- | -------- | ---- | ------ | ---------
+MozFellows Onboarding | Toronto, ON, Canada | 12th-16th September 2016 | :raising_hand: Participant |
 Invited seminar talk | Department of Psychiatry, University of Edinburgh | 21st September 2016 | :loudspeaker: Invited speaker ([slides](https://dx.doi.org/10.6084/m9.figshare.3843405.v3), [abstract/bio](https://github.com/WhitakerLab/WhitakerLabProjectManagement/blob/2b9d52e898549dc43f85b2f93e21dc4c8e22cdde/Kirstie-Whitaker/abstracts_bios_2016.md#invited-talk-in-edinburgh)) | Sue Fletcher-Watson
 [The Educated Brain at School: Late Childhood and Adolescence](http://www.languagesciences.cam.ac.uk/events/the-educated-brain-late-childhood-and-adolescence) | Cambridge, UK | 27th October 2016 | :loudspeaker: Invited speaker ([slides](https://doi.org/10.6084/m9.figshare.4087509.v1), [abstract/bio](https://github.com/WhitakerLab/WhitakerLabProjectManagement/blob/2b9d52e898549dc43f85b2f93e21dc4c8e22cdde/Kirstie-Whitaker/abstracts_bios_2016.md#educated-brain-seminar)) and workshop participant ([policy brief](http://www.publicpolicy.cam.ac.uk/news/educated-brain)) | [Dr Michelle Ellefson](http://www.languagesciences.cam.ac.uk/directory/mre33@cam.ac.uk), Jane Walsh, Dervila Glynn & Charlotte Sausman
 [MozFest](https://mozillafestival.org/) | London, UK | 28th-30th October 2016 | :clipboard: Science space wrangler |
```

Comment:
I feel like you should change this to be Open Science Floor Chair or something because you did a ton of work and "Space Wrangler" really doesn't cover it.